FV3/wrapper/.circleci/config.yml

```diff
@@ -4,14 +4,6 @@ orbs:
   gcp-gcr: circleci/gcp-gcr@0.6.1
```

Comment:
Oh I forgot about this. Yes i'm mostly leaving this as a todo. I can restore the build docs in a follow-up PR since you already took the time to review this pr.

Reply:
Also, I'm worried it could take a lot of debugging to figure this out right now, and there is some question of where the docs shoud go.